website/docs/d/group.html.markdown

```diff
@@ -0,0 +1,50 @@
+---
+layout: "azuread"
+page_title: "Azure Active Directory: azuread_group"
+sidebar_current: "docs-azuread-datasource-azuread-group"
+description: |-
+  Gets information about an Azure Active Directory group.
+
+---
+
+# Data Source: azuread_group
+
+Gets information about an Azure Active Directory group.
+
+-> **NOTE:** If you're authenticating using a Service Principal then it must have permissions to `Read directory data` within the `Windows Azure Active Directory` API.
+
+## Example Usage (by Object ID)
+
+```hcl
+data "azuread_group" "test_group" {
+  object_id = "78722cfc-8946-11e8-95f1-2200ec79ad01"
+}
+```
+
+## Example Usage (by Group Display Name)
+
+```hcl
+data "azuread_group" "test_group" {
+  name = "MyTestGroup"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `object_id` - (Optional) The ID of the Azure AD Group we want to lookup.
+
+* `name` - (Optional) The ID of the Azure AD Group we want to loopup.
+
+-> **NOTE:** At least one of `name` or `object_id` must be specified.
+
+-> **WARNING:** `name` is not unique within Azure Active Directory. The data source will only return the first Group found.
```

Comment:
since this is a warning we can use `~>` rather than `->` (which gives us a yellow box rather than a blue info one)

```suggestion
~> **WARNING:** `name` is not unique within Azure Active Directory. The data source will only return the first Group found.
```

Reply:
made requested changes.